### DISCUSSION TITLE
Russian translation

### DISCUSSION THREAD

**Discussion by vylenne** [nuxt/nuxt#17351] [Ideas]
Hello everybody! I know Russian well, as well as English. If the authors and other users don't mind, then I would make a pool request with a translation.

I hope it will be useful for Russian-speaking developers. 🙌🏼

**Comment by mihail727** [comment]
Я задавал подобный вопрос разработчикам в discord, мне ответили что на данный момент - если я правильно понял - такой возможности нет и они не могут принять помощь

**Reply by vylenne** [reply]
жаль, но если такова их воля - так тому и быть. хотя, скиньте, пожалуйста, дискорд, попробую сама спросить. спасибо.

**Reply by mihail727** [reply]
> жаль, но если такова их воля - так тому и быть. хотя, скиньте, пожалуйста, дискорд, попробую сама спросить. спасибо.

ссылка на дискорд где можно найти всех разработчиков - https://discord.gg/jdZ3t4CbdZ
ник в дискорде Daniel - danielroe#8960
